### DISCUSSION TITLE
404 Github url error

### DISCUSSION THREAD

**Discussion by Ashkansalehpour** [community/community#36491] [Other Feature Feedback, Questions, & Ideas]
I have problem with all of my repositories , I can not create Github url for my repositories and all of my pages have 404 error.

Please someone help me with this error ....

**Comment by radar-blip** [comment]
Here is one of your sites: https://ashkansalehpour.github.io/Personal-Website/index.html  (is working now)

You need to include the docs folder since your index.html is located there.
Either that or bring the .html and .css files out into the main repo.
Easiest way would be to do the same thing you did with your dark-mode site, that seems to be working fine.

Also this link here `<img src="Images/logo.webp" alt="logo" />` (located in your index.html) isn't going to be working because your images folder is lower case.

**Reply by Ashkansalehpour** [reply]
I did all things you said and nothing happen :(

**Reply by radar-blip** [reply]
Both of your personal websites are working, and the NASA one is working too. 
I will look at the others and see if I can figure out the problems.

[Personal](https://ashkansalehpour.github.io/Personal-Website/)

[Personal DarkMode](https://ashkansalehpour.github.io/Personal-Website-DarkMode/)

With Ball-story site, if you can change your `Index.html` to` index.html` that might fix things. (make lowercase)
If that does fix things, do same with Driving-School

**Reply by Ashkansalehpour** [reply]
I changed files name and did all of things you said but it doesn't work :(

**Reply by radar-blip** [reply]
In your Diving School site, I dont think you need this:  ` <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">`  delete if not needed. 

And you should change Style.css (same repo) to match this link:  `<link rel="stylesheet" href="style.css">`
Just change to lowercase.

**Comment by jfullstackdev** [comment]
seems by now, you already have the domain, 

![image](https://user-images.githubusercontent.com/47092464/196479704-b94c4cd2-f999-4ba9-88b3-b30617db9670.png)

but your next problem is the URLs of the images, they cannot be loaded correctly, hopefully you can figure it out, but you can always ask here as you progress

**Comment by ossanna16** [comment]
Hey there @Ashkansalehpour and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by Ashkansalehpour** [comment]
I knew how to fix it , Thanks for your tips to solve this bug . I forgot to change some file name and didn't check the links i used it in my code .
